js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -32,15 +35,16 @@ import './connected-google-combo-account-card.scss';
  * It will also kickoff Ads and Merchant Center account creation if the user does not have accounts.
  */
 const ConnectedGoogleComboAccountCard = () => {
+	const [ editMode, setEditMode ] = useState( false );
+	const { hasDetermined, creatingWhich } = useAutoCreateAdsMCAccounts();
```

Comment:
The `createMCAccount` function should be passed in as a parameter.

![image](https://github.com/user-attachments/assets/a774f529-2f5d-49a6-aea3-b06702c5de20)

Reply:
[0d63a35](https://github.com/woocommerce/google-listings-and-ads/pull/2660/commits/0d63a3591fd22d0dd16a8d4a1a8f16f2c7f47925)